Geometry/HcalCommonData/BuildFile.xml

```diff
@@ -1,6 +1,7 @@
 <use   name="DetectorDescription/Base"/>
 <use   name="DetectorDescription/Core"/>
 <use   name="DataFormats/HcalDetId"/>
+<use   name="SimDataFormats/CaloTest"/>
```

Comment:
@kpedro88 - the dependency should be the other way around - Sim usually depends on Geometry